### PR TITLE
Issue observeruseproxy

### PR DESCRIPTION
**Pull request by FRDavid**
Code for issue #110

### CODE REVIEW

**Comment by TomMettam** [comment]
+1 - I already repaired this in my fork.

**Comment by muoren** [comment]
@FRDavid @TomMettam if you are looking for a more complete implementation of LWM2M, with support for bootstrapping and TLV, JSON, etc. look here: 
* https://github.com/muoren/node-lwm2m

It started as an advanced fork, but not it's a project on its own.
Pull requests, help and ideas welcome, it's still WIP.

File: lib/services/server/informationReporting.js [hunk 1]
```diff
@@ -150,9 +150,11 @@ function processStream(deviceId, objectType, objectId, resourceId, handler, obse
 function observe(deviceId, objectType, objectId, resourceId, handler, callback) {
     function createObserveRequest(obj, callback) {
         var request = {
-            host: obj.address,
```
Comment (fgalan) [inline lib/services/server/informationReporting.js]: Please, add an entry to CHANGES_NEXT_RELEASE file about the issue this PR is fixing, e.g:

```
-Fix: <short description of the bug/issue you are fixing> (#numer_of_issue)
```

Thanks!

**Comment by FRDavid** [comment]
Sorry for the late response, here is the requested change.

**Comment by fgalan** [comment]
LGTM. Thanks!

File: lib/services/server/informationReporting.js (hunk 1 above)
Comment (fgalan) [reply lib/services/server/informationReporting.js]: Fixed in 4c663520c57ebafbcdd8ca463e586b8c28368895

**Pull request merged**